docs/sources/tempo/traceql/metrics-queries/functions.md

```diff
@@ -151,21 +153,27 @@ To demonstrate this flexibility, consider this nonsensical quantile on `span.htt
 { name = "GET /:endpoint" } | quantile_over_time(span.http.status_code, .99, .9, .5)
```

Comment:
```suggestion
{ name = "GET /:endpoint" } | quantile_over_time(span.http.status_code, .99, .9, .5)
This computes the 99th, 90th, and 50th percentile of the values of the `status_code` attribute for all spans named `GET /:endpoint`. This is unlikely to tell you anything useful (what does a median status code of `347` mean?), but it works.
```

Reply:
This may be not helpful. I think you could have a more contrived example like: "Imagine a custom attribute like `span.temperature`. You might want to know what the 50th percentile and 95th percentile temperatures were across all your spans." Even that may seem too nonsensical.

Reply:
Done. I've incorporated your example into the text.